build/repo.targets

```diff
@@ -13,6 +13,7 @@
     <RazorDesignTestProject>$(RepositoryRoot)test\Microsoft.AspNetCore.Razor.Design.Test\</RazorDesignTestProject>
     <MSBuildLocationFileTemplate>$(RazorDesignTestProject)BuildVariables.cs.template</MSBuildLocationFileTemplate>
     <MSBuildLocationFileOutput>$(RazorDesignTestProject)obj\BuildVariables.generated.cs</MSBuildLocationFileOutput>
+    <RazorToolAssembly>$(RepositoryRoot)src\Microsoft.AspNetCore.Razor.Design\bin\$(Configuration)\netstandard2.0\tools\rzc.dll</RazorToolAssembly>
```

Comment:
Is there a better way to get the tool assembly path in tests?

Reply:
Take a look at how the test projects are set up. The build creates the folder structure of `Razor.Design` as it would appear after unpacking the nupkg. I'm not sure this is needed at all.

Reply:
I'll take closer look. Just to be clear I want this value to be accessible in C# in my tests so I can execute shutdown command (Not just setting this value and let MSBuild access it).